csrankings-m.csv

```diff
@@ -2461,7 +2462,6 @@ Mohammad Arif Ul Alam,University of Massachusetts Lowell,http://faculty.uml.edu/
 Mohammad Ashiqur Rahman,Florida International University,https://rahman.eng.fiu.edu/,2OLia-cAAAAJ
 Mohammad Ashjaei,Mälardalen University,http://www.es.mdh.se/staff/360-Mohammad_Ashjaei,AvWVYIkAAAAJ
 Mohammad Atiqul Islam,University of Texas at Arlington,http://crystal.uta.edu/~mislam,rxKWd1cAAAAJ
-Mohammad Atiquzzaman,University of Oklahoma,http://www.cs.ou.edu/~atiq,CPqBDqUAAAAJ

```

Comment:
Explain removal.

Reply:
The faculty's official name is "Mohammed Atiquzzaman" as shown in this page. https://www.ou.edu/coe/cs/people/faculty/mohammed-atiquzzaman
His DBLP entry is also "Mohammed Atiquzzaman"(although with alias name as "Mohammad Atiquzzaman"). https://dblp.org/pid/a/MAtiquzzaman.html
Hence, for the original entry "Mohammad Atiquzzaman", I am not sure this is correct. So I removed it.

Reply:
All DBLP aliases need to be maintained.

Reply:
I see. Thanks. I added back "Mohammad Atiquzzaman", as the new commit [fd269cc](https://github.com/emeryberger/CSrankings/pull/8072/commits/fd269ccff7a376777cb2ccec8bd6f702e4d6470f)